packages/elasticsearch-plugin/src/options.ts

```diff
@@ -222,7 +227,8 @@ export interface ElasticsearchOptions {
     /**
      * @description
      * This config option defines custom mappings which are accessible when the "groupByProduct"
-     * input options is set to `false`.
+     * input options is set to `false`. Anyway, customProductMappings can be accessed by using
+     * customProductMappings field.
```

Comment:
"input options is set to `false`. In addition, custom product mappings can be accessed by using
the `customProductMappings` field, which is always available."